### ISSUE TITLE
Modify career profiles to use salary ranges (2)

### ISSUE THREAD

**Issue by nihonjinrxs** [loyno-mathcs/workforwardnola#7]
As a user considering a potential career path, I would like to see a range of salaries so that I can understand the impact on my future.

**Comment by maahern** [comment]
**BLOCKER:** The careers are managed via a google spreadsheet (https://docs.google.com/spreadsheets/d/1lMeZZ_vu-xIVKLgo7obKlF0X4iJ3oHkLKl8uIrPsSt8, current owners CfA Fellows). -- **Do not have appropriate permissions to update career attributes**

To update the information about a career: edit that spreadsheet, then go to the ["manage" page](http://workforwardnola.com/manage) & enter the username and password. Enter the PUBLIC link to the spreadsheet (https://docs.google.com/spreadsheets/d/1lMeZZ_vu-xIVKLgo7obKlF0X4iJ3oHkLKl8uIrPsSt8/pubhtml), wait for the link to be validated, then click "Update content".

**Comment by nihonjinrxs** [comment]
Per planning session:

- [ ] Database migration to change data type
- [ ] Implement feature

**Comment by maahern** [comment]
Experienced_wage has been removed and replaced with the column experienced_range which is type string. I have edited the HTML to reflect this. This change has not been pushed to the repo as I am waiting on entry wages.

**Comment by bsabo** [comment]
I updated the google spreadsheet that Todd posted to reflect all the salary data. Please let me know if you have any issues with it.

**Comment by maahern** [comment]
Great, thanks! Must've been an updating issue on my end. I'll get started on integrating entry wages onto the job results page.

**Comment by maahern** [comment]
Pull request has been made. Ready to deploy.

**Comment by nihonjinrxs** [comment]
@maahern when you all make a PR relevant to an issue, please reference it in the issue via it's number. Any number with a # in front of it will be auto-linked to the relevant issue or PR in this repo.

cc @loyno-mathcs/f2017-software-engineering

**Comment by maahern** [comment]
Oh ok, good to know!

**Comment by tmloupe** [comment]
Complete and merged into master.
